etc/module.xml

```diff
@@ -18,7 +18,7 @@
 -->
 <!-- Bolt Payment Module Config. Plugin version and dependency loading sequence. -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
-    <module name="Bolt_Boltpay" setup_version="2.2.0">
+    <module name="Bolt_Boltpay" setup_version="2.2.0.1">
```

Comment:
we dont do these versions anymore. only semver.

Reply:
@johnmccombs1 Is this leftover from testing on their server? We shouldn't be merging changes to versions outside of version bump PRs unless it's a custom branch.

Reply:
@johnmccombs1 I'm guessing you did this so that `upgradeSchema` would make your table, but I think we should avoid this practice. We'll version bump 2.3.0 later this week to get the schema created.